pay-api/tests/unit/models/test_fee_schedule.py

```diff
@@ -190,104 +190,3 @@ def get_or_create_corp_type(corp_type_code: str, corp_description: str, session)
     if existing:
         return existing
     return CorpType(code=corp_type_code, description=corp_description)
-
-
-def test_get_fee_details_all_products(session):
```

Comment:
These queries have been moved to the service, bye bye model tests.

Already tested as API and Service tests